### PR TITLE
feat: Center modal on mobile & add fullscreen option on mobile

### PR DESCRIPTION
**Pull request by GoOz**
- [x] Modal are now vertically centered on mobile (<768px)
- [x] Added `fullscreen` boolean props to allow a Modal to fit the entire screen on mobile (<768px) if needed (default `false`), mostly for Modals with a lot of content.

### CODE REVIEW

**Comment by ptbrowne** [comment]
If the fullscreen options only works on mobile it should be named accordingly IMHO : `mobileFullscreen` ? Can you build and push the styleguide :) ?

**Comment by GoOz** [comment]
https://gooz.github.io/cozy-ui/react/#modal

**Comment by GoOz** [comment]
[mobile modales](https://trello.com/c/RRq33oxb/122-mobile-modales)

**Comment by GoOz** [comment]
oh good catch! no it's not normal.

File: react/Modal/Readme.md [hunk 1]
```diff
@@ -66,6 +66,21 @@ const spacings = [
 </div>
 ```
 
+### mobileFullscreen
+
+If you have a modal filled with content and want it to occupy all the available space on mobile screen.
```
Comment (ptbrowne) [inline react/Modal/Readme.md]: If you want the modal to fill all the available space, without margin, on mobile screen.

**Pull request merged**